tests/bluetooth/shell/audio.conf

```diff
@@ -21,6 +21,10 @@ CONFIG_BT_HRS=y
 CONFIG_BT_WHITELIST=y
 CONFIG_BT_REMOTE_INFO=y
 CONFIG_BT_REMOTE_VERSION=y
+CONFIG_BT_CTLR_ADV_EXT=y
+CONFIG_BT_CTLR_ADV_PERIODIC=y
+CONFIG_BT_CTLR_SYNC_PERIODIC=y
+
```

Comment:
Move this to board specific conf file, i.e. `tests/bluetooth/shell/boards/nrf52840dk_nrf52840.conf`

Reply:
Done - for both 52840 and 5340